flatland_server/src/model.cpp

```diff
@@ -162,6 +164,14 @@ ModelBody *Model::GetBody(const std::string &name) {
   return nullptr;
 }
 
+World *Model::GetWorld() {
+  return world;
+}
+
+MessageServer *Model::GetMessageServer() {
+    return &(GetWorld()->message_server);
```

Comment:
So can the world be null or not?

Reply:
No, the world contains the message server, it is impossible for the message server to exist without the world

Reply:
Then why don't you return a reference instead of a pointer for both GetWorld and GetMessageServer?

Reply:
Sure